Sources/SwifterSwift/UIKit/UITextFieldExtensions.swift

```diff
@@ -144,18 +144,44 @@ public extension UITextField {
         leftViewMode = .always
     }
 
+    /// SwifterSwift: Add padding to the right of the textfield rect.
+    ///
+    /// - Parameter padding: amount of padding to apply to the right of the textfield rect.
+    func addPaddingRight(_ padding: CGFloat) {
+        let paddingView = UIView(frame: CGRect(x: 0, y: 0, width: padding, height: frame.height))
+        rightView = paddingView
```

Comment:
There's no need to do this in 2 steps. Also, you should be using `bounds`, not `frame`.
```suggestion
        rightView = UIView(frame: CGRect(x: 0, y: 0, width: padding, height: bounds.height))
```
Feel free to fix the function above too.